### PR TITLE
PUBDEV-8505 Document make_metrics method

### PR DESCRIPTION
**Pull request by hannah-tillman**
For: [PUBDEV-8505](https://h2oai.atlassian.net/browse/PUBDEV-8505)

I placed this information in the [evaluation model metrics section](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/performance-and-prediction.html#evaluation-model-metrics) after the classification & regression sections but before their best practices since it works for both classification & regression problems. Please let me know if anything needs to be updated!

### CODE REVIEW

File: h2o-docs/src/product/performance-and-prediction.rst [hunk 1]
```diff
@@ -1163,6 +1163,87 @@ Using the previously imported and split airlines dataset, run the following to r
     ks
     0.20072346203696562
 
+Computing Model Metrics
+'''''''''''''''''''''''
+
+The ``h2o.make_metrics`` function computes a model metrics object from given predicted values (target for regression; class-1 probabilities, binomial, or per-class probabilities for classification).
```
Comment (maurever) [inline h2o-docs/src/product/performance-and-prediction.rst:1169]: The ``h2o.make_metrics`` is R specific name, maybe only ``make_metrics`` is general enough, WDYT? :) 

```suggestion
The ``make_metrics`` function computes a model metrics object from given predicted values (target for regression; class-1 probabilities, binomial, or per-class probabilities for classification). 
```

File: h2o-docs/src/product/performance-and-prediction.rst [hunk 2]
```diff
@@ -1163,6 +1163,87 @@ Using the previously imported and split airlines dataset, run the following to r
     ks
     0.20072346203696562
 
+Computing Model Metrics
```
Comment (maurever) [inline h2o-docs/src/product/performance-and-prediction.rst:1166]:
```suggestion
Computing Model Metrics from general predictions
```

WDYT @ledell ?

Comment (ledell) [reply h2o-docs/src/product/performance-and-prediction.rst:1166]: I think that's good, thanks @maurever.

**Review by ledell — APPROVED** [review]
This looks great, thanks @hannah-tillman!

**Review by maurever — APPROVED** [review]
Thanks @hannah-tillman!

**Pull request merged**